### DISCUSSION TITLE
Masque not Showing GCDs

### DISCUSSION THREAD

**Discussion by Tinidan** [SFX-WoW/Masque#187] [Issues] [answered]
Masque not showing GCDs when I pressed buttons

**Comment by StormFX** [comment, marked as answer]
This really doesn't tell me anything. Masque only skins what's passed to it. If your GCDs aren't displaying, check the options for your action bar add-on.

**Comment by StormFX** [comment]
This is working fine with Bartender4 and Dominos.
